src/mesh/generated/meshtastic/mesh.pb.h

```diff
@@ -694,7 +694,9 @@ typedef struct _meshtastic_MeshPacket {
     /* Describes whether this packet passed via MQTT somewhere along the path it currently took. */
     bool via_mqtt;
     /* Hop limit with which the original packet started. Sent via LoRa using three bits in the unencrypted header.
- When receiving a packet, the difference between hop_start and hop_limit gives how many hops it traveled. */
+ When receiving a packet, the difference between hop_start and hop_limit gives how many hops it traveled.
+ 
+ note: this value is not present for firmware < 2.3.x */
```

Comment:
comment not accurate, it will always be present, but will always be zero for old nodes